programs/compressed-token/src/process_transfer.rs

```diff
@@ -243,19 +241,20 @@ pub struct TransferInstruction<'info> {
     pub self_program: Program<'info, crate::program::PspCompressedToken>,
 }
 
-// TODO: parse utxos a more efficient way, since owner is sent multiple times this way
+// TODO: parse compressed_accounts a more efficient way, since owner is sent multiple times this way
 // This struct is equivalent to the InstructionDataTransfer, but uses the imported types from the psp_compressed_pda
 #[derive(Debug, Clone, AnchorSerialize, AnchorDeserialize)]
 pub struct InstructionDataTransfer {
```

Comment:
if we give this struct the same name as in compress-pda, we break the IDL. 
(previously it just didn't break because compressed-pda was defining its InstructionDataTransfer struct as account not as type)
In my feature branch,  I've temporarily renamed this to CompressedTokenInstructionDataTransfer in order to fix the IDL. there's probably a better name for it tho.